From 8e6a0c632739f323103fb66720d832b431eafaeb Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Sun, 30 Apr 2017 03:55:47 +0100
Subject: [PATCH 2/2] kernel-modules: Always include hid-logitech-hidpp

This driver should be preloaded into the initrd to ensure that users
with wireless Logitech keyboards are able to utilise them during the
dm-crypt unlock of an LVM/LUKS rootfs.

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 modules.d/90kernel-modules/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90kernel-modules/module-setup.sh b/modules.d/90kernel-modules/module-setup.sh
index a7b0853..396caf4 100755
--- a/modules.d/90kernel-modules/module-setup.sh
+++ b/modules.d/90kernel-modules/module-setup.sh
@@ -43,7 +43,7 @@ installkernel() {
 
         instmods yenta_socket scsi_dh_rdac scsi_dh_emc scsi_dh_alua \
             atkbd i8042 usbhid hid-apple hid-sunplus hid-cherry hid-logitech \
-            hid-logitech-dj hid-microsoft hid-lcpower firewire-ohci \
+            hid-logitech-dj hid-logitech-hidpp hid-microsoft hid-lcpower firewire-ohci \
             pcmcia hid-hyperv hv-vmbus hyperv-keyboard
 
         if [[ "$(uname -p)" == arm* ]]; then
-- 
2.12.2

